pulsar-broker/src/test/java/org/apache/pulsar/broker/service/OneWayReplicatorTest.java

```diff
@@ -698,10 +708,19 @@ public void testDeleteNonPartitionedTopic() throws Exception {
         // Verify replicator works.
         verifyReplicationWorks(topicName);
 
+        admin1.topicPolicies().setCompactionThreshold(topicName, 1000_000);
+        Awaitility.await().atMost(Duration.ofSeconds(3600)).untilAsserted(() -> {
```

Comment:
I see this check will retry many times, and it takes a very long time, is this expected?

Reply:
Sorry, it is for debugging, I removed it.